<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1386</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>571</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>МАСКА ВХОДНЫХ ФАЙЛОВ</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_1">
         <property name="text">
          <string>*.*</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ПУТЬ К ВЫХОДНЫМ ФАЙЛАМ </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="text">
          <string>C:/Users/artem/Desktop/F/</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>ПЕРИОД ОПРОСА НАЛИЧИЯ ФАЙЛА</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>МАСКА МОДИФИКАЦИИ ФАЙЛА</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>ДЕЙСТВИЕ</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>70</y>
      <width>440</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_1">
       <property name="title">
        <string>УДАЛЕНИЕ ВХОДНОГО ФАЛА</string>
       </property>
       <widget class="QRadioButton" name="radioButton_1_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>112</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>ДА</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_1_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>112</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>НЕТ</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>ДЕЙСТВИЕ ПРИ ПОВТОРЕНИИ ИМЕНИ ВХОДНОГО ФАЙЛА</string>
       </property>
       <widget class="QRadioButton" name="radioButton_2_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>565</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>ПЕРЕЗАПИСАТЬ</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>565</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>ДОБАВИТЬ С НОВЫМ ИМЕНЕМ</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>РЕЖИМ РАБОТЫ</string>
       </property>
       <widget class="QRadioButton" name="radioButton_3_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>158</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>РАЗОВЫЙ ЗАПУСК</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_3_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>122</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>ПО ТАЙМЕРУ</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>370</y>
      <width>441</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1386</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
